cogs/team.py

```diff
@@ -234,6 +237,36 @@ async def reject_callback(interaction: discord.Interaction):
                 + "Tag an exec and they can add the member to your team for you!"
             )
 
+    @app_commands.command(name="info", description="Check your team status.")
+    @in_team_channel
+    async def team_info(self, interaction: discord.Interaction):
+        await interaction.response.defer()
+
+        player = await player_query.get_player(interaction.user.id)
+        team = await team_query.get_team(player.team_name)
+
+        info_embed = discord.Embed(
+            colour=discord.Color.random(),
+            title=f"{team.team_name} Info",
+        )
+
+        accessible_puzzles = await get_accessible_puzzles(team.team_name)
+        info_embed.add_field(
+            name="Puzzle Status",
+            value=f"You have solved {team.puzzle_solved} puzzles "
+            + f"out of {len(accessible_puzzles)} available!",
+        )
+
+        max_hints = await get_max_hints()
+        next_hint_time = await get_next_hint_time()
+        info_embed.add_field(
+            name="Hint Status",
+            value=f"You have used {team.hints_used} out of {max_hints} available. "
+            + f"Next hint at {next_hint_time}.",
+        )
```

Comment:
does this logic need to have a case of when 3 teams have finished -unlimited hint?